src/SFA.DAS.LevyTransferMatching.Web/Controllers/SearchFunding.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+
+namespace SFA.DAS.LevyTransferMatching.Web.Controllers
+{
+    [AllowAnonymous]
+    public class SearchFunding : Controller
+    {
+        [Route("search-funding-opportunities")]

```

Comment:
We can probably leave the route empty for this, so that it is accessed from `/search-funding`